config/locales/uk.yml

```diff
@@ -28,13 +28,15 @@ uk:
       name: "Ім'я"
       links: "Функції"
       choose_file: "Оберіть файл"
+      file_name: "Ім'я файлу:"
```

Comment:
can not find some translations for these keys in the en.yml file. Is this wording not translatable?

Reply:
Added new translations